docs/topics/classes.md

```diff
@@ -101,7 +101,7 @@ Learn more about [visibility modifiers](visibility-modifiers.md#constructors).
 The class can also declare _secondary constructors_, which are prefixed with `constructor`:
 
 ```kotlin
-class Person {
+class Person() {
```

Comment:
Thanks, this example was a bit incorrect and we've already reworked it. Check the new one [here](https://kotlinlang.org/docs/classes.html#secondary-constructors).

Please note that the changes you suggest also have an issue: 
`constructor(parent: Person) {` doesn't work without a primary constructor call (see the paragraph after this example).